e2e/global/common/integration_fail_test.go

```diff
@@ -62,3 +61,28 @@ func TestBadRouteIntegration(t *testing.T) {
 		Expect(Kamel("delete", "--all", "-n", ns).Execute()).To(Succeed())
 	})
 }
+
+func TestMissingDependencyIntegration(t *testing.T) {
```

Comment:
I don't see a reason to separate the namespaces for the test. Can't we merge them into a single test func?

Reply:
Yeah, probably it was a better decision. This is already merged, I'll see if I can open a new PR with the fix.

Reply:
Fixed in [22a79ae](https://github.com/squakez/camel-k/commit/22a79ae41b6fcbf4dc781110a4b3849fb244ed5e)